src/main/java/org/janelia/saalfeldlab/paintera/PainteraDefaultHandlers.java

```diff
@@ -465,10 +465,11 @@ public static void grabFocusOnMouseOver(final Node node)
 
 	public void toggleInterpolation()
 	{
-		final Source<?> source = sourceInfo.currentSourceProperty().get();
-		if (source == null) { return; }
-		final ObjectProperty<Interpolation> ip = sourceInfo.getState(source).interpolationProperty();
-		ip.set(ip.get().equals(Interpolation.NLINEAR) ? Interpolation.NEARESTNEIGHBOR : Interpolation.NLINEAR);
+		for (final Source<?> source : sourceInfo.trackSources())
```

Comment:
This only makes sense if there is a global interpolation state. Think of this scenario:
Add source1, toggle interpolation (source1 is now in `NLINEAR`).
Add source2, toggle interpolation (source1 is now in `NEARESTNEIGHBOR`, source2 is now in `NLIENAR`).

Reply:
Good point, I'll look into changing interpolation mode to be a global state instead.

Reply:
We should keep the source specific interpolation (but control it through the global state for now), in case we want to be able to add an option to set interpolation individually per source